preload readline

cielavenir opened the pull request:
On rundocker.py envioronment, "ipython with viewer" will crash without this patch.

cielavenir commented:
(I'm expecting Kou to join also as reviewer)

cielavenir commented:
workaround: `function rave () { DISPLAY=:0 python -m IPython -c "import os,sys;sys.argv=sys.argv[2:];rave=open('/usr/local/bin/openrave.py').read();exec(rave);exit()" "$@"; }`

cielavenir commented:
I just found that this IPython matter is affected IPython<=4.2.1. If we move to IPython 5.x, this does not matter anymore.

rdiankov commented:
great~ i can accept this patch if you comment out the line and document that it is necessary for ipython <= 4.2.1. 
thanks~

cielavenir commented:
maybe I did not understand correctly - "commenting out" will require uncommenting to enable the feature. namely modifying code. It the same as adding the code after every satrtup.

----

Now I change the method. I removed preloading and moved importing IPython before creating viewer. Is this ok?
